Balloon/sys/cleanAll.bat

```diff
@@ -2,5 +2,13 @@
 
 for /d %%x in (objfre_*) do rmdir /S /Q %%x
 for /d %%x in (objchk_*) do rmdir /S /Q %%x
+rmdir /S /Q .\sdv
+rmdir /S /Q .\sdv.temp
 
 del /F *.log *.wrn *.err
+
+del *.dvl.xml
```

Comment:
Please also remove *.dvl-compat.xml. We create *.dvl-compat.xml for compatibility with Windows 8.

Reply:
Done